codesign/test/file_codesign_visitor_test.dart

```diff
@@ -20,33 +19,29 @@ void main() {
   const String appSpecificPasswordFilePath = '/tmp/passwords.txt';
   const String codesignAppstoreIDFilePath = '/tmp/appID.txt';
   const String codesignTeamIDFilePath = '/tmp/teamID.txt';
+  const String inputZipPath = '/tmp/input.zip';
+  const String outputZipPath = '/tmp/output.zip';
   final MemoryFileSystem fileSystem = MemoryFileSystem.test();
   final List<LogRecord> records = <LogRecord>[];
 
   late FakeProcessManager processManager;
-  late GoogleCloudStorage googleCloudStorage;
   late cs.FileCodesignVisitor codesignVisitor;
 
   Directory rootDirectory = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
 
   group('test reading in passwords: ', () {
     setUp(() {
       processManager = FakeProcessManager.list(<FakeCommand>[]);
-      googleCloudStorage = GoogleCloudStorage(
-        processManager: processManager,
-        rootDirectory: rootDirectory,
-      );
       codesignVisitor = cs.FileCodesignVisitor(
         codesignCertName: randomString,
-        googleCloudStorage: googleCloudStorage,
         fileSystem: fileSystem,
         appSpecificPasswordFilePath: appSpecificPasswordFilePath,
         codesignAppstoreIDFilePath: codesignAppstoreIDFilePath,
         codesignTeamIDFilePath: codesignTeamIDFilePath,
         processManager: processManager,
         rootDirectory: rootDirectory,
-        gcsDownloadPath: 'gs://flutter/$randomString/$randomString',
-        gcsUploadPath: 'gs://flutter/$randomString/$randomString',
+        inputZipPath: randomString,
```

Comment:
do we really want the input and output path to be the same? also, shouldn't these files have a .zip extension?

Reply:
Very true I meant to set them to inputZipPath and outputZipPath. Good eye Christopher!